docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -2649,6 +2649,71 @@ class DetailResource {
 
 WARNING: Unlike with `@Inject` the templates obtained via `RestTemplate` are not validated, i.e. the build does not fail if a template does not exist.
 
+[[vertx_integration]]
+=== Vertx Integration
+
+If you want to use `io.vertx.core.json.JsonObject` as data in your templates, then you will need to add the `quarkus-vertx` extension to your `pom.xml` file (most applications use this extension by default).
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-vertx</artifactId>
+</dependency>
+----
```

Comment:
Here we should use the tabs with Gradle option too. There are numerous examples in the doc (for instance in `validation.adoc`).

Reply:
And the same applies here - remove the GAV.

Reply:
> And the same applies here - remove the GAV.

Why remove GAV @mkouba ?